src/test/java/org/alfresco/rest/api/tests/client/data/SiteGroup.java

```diff
@@ -0,0 +1,191 @@
+/*
+ * #%L
+ * Alfresco Remote API
+ * %%
+ * Copyright (C) 2005 - 2016 Alfresco Software Limited
+ * %%
+ * This file is part of the Alfresco software.
+ * If the software was purchased under a paid Alfresco license, the terms of
+ * the paid license agreement will prevail.  Otherwise, the software is
+ * provided under the following open source license terms:
+ *
+ * Alfresco is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Alfresco is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Alfresco. If not, see <http://www.gnu.org/licenses/>.
+ * #L%
+ */
+
+package org.alfresco.rest.api.tests.client.data;
+
+import org.alfresco.rest.api.tests.client.PublicApiClient;
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+
+import java.io.Serializable;
+import java.text.Collator;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+public class SiteGroup implements Serializable, ExpectedComparison, Comparable<SiteGroup> {
+	private static final long serialVersionUID = 505331886661880399L;
+
+	private String role;
+	private String id; // group id (aka authority name)
+	private Group group;
+
+	public SiteGroup() {
+		super();
+	}
+
+	public SiteGroup(String id, String role) {
+		this.role = role;
+		this.id = id;
+	}
+
+	public SiteGroup(String id, Group group, String role) {
+		this.role = role;
+		this.id = id;
+		this.group = group;
+	}
+
+	public String getRole() {
+		return role;
+	}
+
+	public void setRole(String role) {
+		this.role = role;
+	}
+
+	public String getId() {
+		return id;
+	}
+
+	public void setId(String id) {
+		this.id = id;
+	}
+
+	public Group getGroup() {
+		return group;
+	}
+
+	public void setGroup(Group group) {
+		this.group = group;
+	}
+
+	public Collator getCollator() {
+		return collator;
+	}
+
+	public void setCollator(Collator collator) {
+		this.collator = collator;
+	}
+
+	public static SiteGroup parseSiteGroup(String siteId, JSONObject jsonObject)
+	{
```

Comment:
In this file sometimes the braces are on a new line others they are not.
Maybe, for this project, we should follow the coding standard defined here https://hub.alfresco.com/t5/alfresco-content-services-hub/coding-standards-for-alfresco-content-services/ba-p/290457